models/purchase.py

```diff
@@ -0,0 +1,28 @@
+from datetime import datetime, timedelta
+from odoo import models, fields, api
+from odoo.exceptions import UserError
+
+class PurchaseOrder(models.Model):
+    _inherit = 'purchase.order'
+
+    active = fields.Boolean(default=True)
+    lifespan = fields.Integer(default=0)
```

Comment:
Add a new setting to _**Purchase > Configurations > Settings**_ instead of adding it for each Purchase Order because we'll have one `lifespan` for all of our purchase orders